reporter.go

```diff
@@ -222,3 +224,9 @@ func logFileFor(dirName string, testName string, kind string) (*os.File, error)
 	}
 	return f, nil
 }
+
+func cleanDirName(dirName string) string {
+	res := strings.ReplaceAll(dirName, "/", "|")
```

Comment:
I'd avoide pipe in paths. If nothing else, requires more quoting. What about
```
res := strings.ReplaceAll(dirName, "/", "-")
res = strings.ReplaceAll(res, " ", "_")
```

Reply:
no problem I'll change that.
feel free to let me know if there are any other chars you think need to be replaced

Reply:
done